ebookstore-ui/src/SignIn.tsx

```diff
@@ -123,7 +123,7 @@ export default function SignIn() {
                                 required
                                 fullWidth
                                 id="email"
-                                label="Email Address"
+                                label="Email Address / User Name"
                                 name="email"
                                 autoFocus
                                 autoComplete="off"
```

Comment:
This text box allows non email entries as well. We are using a plain text box for entering email and i don't think we have any field level validations in place in the front-end.  Currently we have added some validations in the registration page but we don't have any related validation in place to avoid non email entries.